solr_conf_4_testing/schema.xml

```diff
@@ -235,8 +235,6 @@
     <dynamicField name="*_facet" type="string" stored="true" indexed="true" multiValued="true" />
     <dynamicField name="*_display" type="string" stored="true" indexed="false" multiValued="true"/>
 -->
-    <dynamicField name="*_si" type="string" stored="true" indexed="true" omitNorms="true" />
```

Comment:
certainly not ALL of them.  But i figured if the exhibits/spotlight/ indexing  code didn't have it, it probably wouldn't be in any of them, and confirmed with 2 or 3 exhibits' Solr schema browser.